Failing test on Node v6

Issue seems to stem from hdr histogram. Testing as appropriate, will report findings after.

Output:

```sh
Glens-MacBook-Pro-2:autocannon glen$ npm run test

> autocannon@0.5.2 test /Users/glen/work/nearform/os/autocannon
> standard && tap test/*.test.js

test/format.test.js ................................... 4/4
test/myhttp.test.js ................................. 29/29
test/run.test.js .................................... 47/48
  run
  not ok throughput.stddev exists
    at:
      line: 34
      column: 7
      file: test/run.test.js
    stack: |
      test/run.test.js:34:7
      Timeout.setInterval (lib/run.js:100:7)
    source: |
      t.ok(result.throughput.stddev, 'throughput.stddev exists')

total ............................................... 80/81


  80 passing (5s)
  1 failing


npm ERR! Darwin 15.5.0
npm ERR! argv "/Users/glen/.nvm/versions/node/v6.2.2/bin/node" "/Users/glen/.nvm/versions/node/v6.2.2/bin/npm" "run" "test"
npm ERR! node v6.2.2
npm ERR! npm  v3.9.5
npm ERR! code ELIFECYCLE
npm ERR! autocannon@0.5.2 test: `standard && tap test/*.test.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the autocannon@0.5.2 test script 'standard && tap test/*.test.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the autocannon package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     standard && tap test/*.test.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs autocannon
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls autocannon
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/glen/work/nearform/os/autocannon/npm-debug.log
```

--- a/test/run.test.js
+++ b/test/run.test.js
@@ -31,7 +31,7 @@ test('run', (t) => {
 
     t.ok(result.throughput, 'throughput exists')
     t.ok(result.throughput.average, 'throughput.average exists')
-    t.ok(result.throughput.stddev, 'throughput.stddev exists')
+    t.type(result.throughput.stddev, 'number', 'throughput.stddev exists')
     t.ok(result.throughput.min, 'throughput.min exists')
     t.ok(result.throughput.max, 'throughput.max exists')
     t.ok(result.throughput.total >= result.throughput.average * 2 / 100 * 95, 'throughput.total exists')
@@ -71,7 +71,7 @@ test('tracker.stop()', (t) => {
 
     t.ok(result.throughput, 'throughput exists')
     t.ok(result.throughput.average, 'throughput.average exists')
-    t.ok(result.throughput.stddev, 'throughput.stddev exists')
+    t.type(result.throughput.stddev, 'number', 'throughput.stddev exists')
     t.ok(result.throughput.min, 'throughput.min exists')
     t.ok(result.throughput.max, 'throughput.max exists')
     t.ok(result.throughput.total >= result.throughput.average * 2 / 100 * 95, 'throughput.total exists')

